src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -656,6 +656,7 @@ def test_cosmosdb_sql_container_update_analytical_ttl(self, resource_group):
         container_list = self.cmd('az cosmosdb sql container list -g {rg} -a {acc} -d {db_name}').get_output_in_json()
         assert len(container_list) == 0
 
+    @unittest.skip('Requests to disable analytics temporarily blocked in production. Will reenable test once disable analytics capability is restored.')
```

Comment:
Instead of skipping the test, I'll recommend to mark this test as `record_only` so that it won't be executed in live mode. No real traffic would happen. You can remove the `record_only` decorator after service restored

Reply:
Thanks for the recommendation. Made the change @evelyn-ys please review again